### PR TITLE
Add polling for STARTED games on Frontend

### PR DESCRIPTION
**Pull request by iva-stoycheva**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -11,3 +11,9 @@ export const sendHeartbeat = async (user) => {
   }
   return await fetch('/heartbeat', requestOptions)
 }
+
+export const getStartedGame = async (playerId) => {
+  const response = await fetch(`/my-started-game?playerId=${playerId}`)
```
Comment (SlavenGrowy) [inline src/api.js:16]: This seems to be okay in the frontend, but calling the endpoint in the backend returns a 500:
```
Error while getting started games 😬 TypeError: Cannot read properties of undefined (reading 'Items')
    at Dynamo.getStartedGames (file:///Users/slavi/git/tic-tac-toe/tic-tac-toe-backend/src/dynamo.js:72:32)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///Users/slavi/git/tic-tac-toe/tic-tac-toe-backend/src/app.js:61:26
```
Does this happen on your end as well?

Comment (SlavenGrowy) [reply src/api.js:16]: Also what happens when our endpoint returns 404?

File: src/constants.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export const userFetchInterval = 3 * 1000
 export const heartbeatInterval = 15 * 1000
+export const gameInterval = 3 * 1000
```
Comment (SlavenGrowy) [inline src/constants.js:3]: A better name might be `myGameFetchInterval`

File: src/pages/Home.js [hunk 3]
```diff
@@ -18,11 +18,24 @@ export default function Home() {
     }
   }
 
+  const navigateToStartedGame = () => {
```
Comment (SlavenGrowy) [inline src/pages/Home.js:21]: A more descriptive name for this function might be: `pollForStartedGame`

**Review by SlavenGrowy — CHANGES_REQUESTED** [review]
Good start 🙌

File: src/api.js (hunk 1 above)
Comment (iva-stoycheva) [reply src/api.js:16]: We don't get this error and also when our endpoint returns 404, it sends 'Not found started games 😬'.

File: src/api.js [hunk 4]
```diff
@@ -11,3 +11,12 @@ export const sendHeartbeat = async (user) => {
...
+
+  if (response.status === 404) console.log('NOT FOUND started game')
```
Comment (SlavenGrowy) [inline src/api.js:18]: This is a good start, but when we get a 404 response our body is empty, so the lines below will cause an Error to be thrown.

**Review by SlavenGrowy — CHANGES_REQUESTED** [review]
Almost there :)

**Review by SlavenGrowy — CHANGES_REQUESTED** [review]

**Review by SlavenGrowy — APPROVED** [review]
Good stuff 🚀🎆💥

**Pull request merged**